packages/nameguard-sdk/src/index.ts

```diff
@@ -682,6 +682,8 @@ export class NameGuard {
    *
    * Returns `display_name` to be shown to users and estimates `impersonation_status`
    *
+   * If `options.computeNameGuardReport` is `False`, the `nameguard_result` field will be `null`.
```

Comment:
@djstrong I think it's better to invert the way this idea is documented. This way we also (implicitly) define what happens if this param is undefined. For example:

```suggestion
   * If `address` has a primary name and `options.computeNameGuardReport` is `True`, then NameGuard will attempt to inspect that primary name. Else, NameGuard will not perform any primary name inspection and the returned  `nameguard_result` field  will be `null`.
```